app/controllers/activities_controller.rb

```diff
@@ -1,17 +1,33 @@
 class ActivitiesController < ApplicationController
   PAR_PAGE = 10
 
+  before_action :set_target_date
+
   def index
-    @activities = Activity.related_user(current_user).
+    @activities = Activity.
+      related_user(current_user).
+      between_beginning_and_end_of_day(@target_date).
       default_order.
-      page(params[:page]).per(PAR_PAGE)
+      page(params[:page]).
+      per(PAR_PAGE)
   end
 
   def all
-    @activities = Activity.joins(:notice, :user).
+    @activities = Activity.
+      joins(:notice, :user).
+      between_beginning_and_end_of_day(@target_date).
       default_order.
-      page(params[:page]).per(PAR_PAGE)
-
+      page(params[:page]).
+      per(PAR_PAGE)
     render "index"
   end
+
+  private
+    def set_target_date
```

Comment:
Assignment Branch Condition size for set_target_date is too high. [16.31/15]<br>Inconsistent indentation detected.